README.md

```diff
@@ -95,6 +95,10 @@ git clone https://github.com/Dasharo/dcu.git
 
 Simply use `dcuc` instead od `dcu`, and follow the section below.
 
+NOTE: to avoid errors when using container version or tests that use dcu,
```

Comment:
Please link this issue here for details: https://github.com/Dasharo/dcu/issues/17

We need to resolve it somehow later on